docs/install-on-microk8s.md

```diff
@@ -0,0 +1,49 @@
+# Install on Minikube
+
+## Pre-requisites
+- [MicroK8s](https://microk8s.io/)
+- To install etcd using these instructions, you must have [Helm](https://helm.sh) in your client environment.
+- [Download the Compose on Kubernetes installer](https://github.com/docker/compose-on-kubernetes/releases).
+- Ensure storage and dns are enabled: `microk8s.enable storage dns`
+- Alias kubectl from the snap with: `sudo snap alias microk8s.kubectl kubectl`.

```

Comment:
It is only required if you don't have kubectl already. (as the procedure make it copy the microk8s kubeconfig in ~/.kube/config)